assets/js/gamepage.js

```diff
@@ -70,9 +70,25 @@ centerThemeCard.on('click', function(event) {
   });
 });
 
-function endGame(remaining) {
+function saveHighScores() {
+  // store multiple objects in an array, this will help with storing all of your highscore objects together
+  var username = localStorage.getItem("user_name");
+  var score = localStorage.getItem("score");
+
+  var highscore = {
+    username: username,
+    score: score
+  }
```

Comment:
If you want to save more than one highscore you will need to create an array  and push the current high score onto the array before you save to local storage. If you want an example of what 'm talking about DM me in slack and I can talk it over with you